frontend/src/sass/pages/_permit-guidelines.scss

```diff
@@ -8,7 +8,7 @@
   //     padding-top: 60px;
   //   }
   // }
-}
+//}
 
```

Comment:
Can we clean up / remove any commented code

Reply:
Commented code removed